UnitsNet/UnitDefinitions/Magnetization.json

```diff
@@ -0,0 +1,20 @@
+﻿{
+  "Name": "Magnetization",
```

Comment:
Is Magnetization the correct naming?
I see magnetic field and flux density being referenced [here](http://www.hkd.co.jp/english/unit_conv/), but I don't know anything about these units.

Reply:
https://en.wikipedia.org/wiki/Magnetization

It's an important parameter for some devices found in electric substations.